numpy/core/tests/test_multiarray.py

```diff
@@ -6305,7 +6249,6 @@ def test_matmul_bool(self):
         assert not np.any(c)
 
 
-if sys.version_info[:2] >= (3, 5):
```

Comment:
You need to unindent the lines below this `if`

Reply:
whoops, thanks for the 2nd pair of eyes.